tests/components/roborock/test_init.py

```diff
@@ -12,7 +12,7 @@
 
 async def test_unload_entry(hass: HomeAssistant, bypass_api_fixture) -> None:
     """Test unloading roboorck integration."""
-    entry = await setup_platform(hass, Platform.VACUUM)
+    entry = await setup_platform(hass, [Platform.VACUUM])
```

Comment:
We should not set up specific platforms, but set up the integration as normal (with all platforms).

Reply:
changed to setup_platforms and removed the platform patch.